blockchain/types/anchoring_data.go

```diff
@@ -18,20 +18,33 @@ package types
 
 import (
 	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/ser/rlp"
 	"math/big"
 )
 
-type ChainHashes struct {
+type AnchoringData struct {
+	Type uint8
+	Data []byte
+}
+
+type AnchoringDataInternalType0 struct {
 	BlockHash     common.Hash
 	TxHash        common.Hash
 	ParentHash    common.Hash
 	ReceiptHash   common.Hash
 	StateRootHash common.Hash
 	BlockNumber   *big.Int
+	Period        *big.Int
+	TxCount       *big.Int
 }
 
-func NewChainHashes(block *Block) *ChainHashes {
-	return &ChainHashes{block.Hash(), block.Header().TxHash,
+func NewAnchoringDataType0(block *Block, period *big.Int, txCount *big.Int) (*AnchoringData, error) {
+	data := &AnchoringDataInternalType0{block.Hash(), block.Header().TxHash,
 		block.Header().ParentHash, block.Header().ReceiptHash,
-		block.Header().Root, block.Header().Number}
+		block.Header().Root, block.Header().Number, period, txCount}
+	encodedCCTxData, err := rlp.EncodeToBytes(data)
+	if err != nil {
+		return nil, err
+	}
+	return &AnchoringData{0, encodedCCTxData}, nil
```

Comment:
How about declaring a const variable for type 0. 
It will help to find where the type value is used.

Reply:
As discussed earlier, type is virtually version number for the anchoring tx. I think it is not necessary to define "`const TYPE0 = 0`".

https://github.com/klaytn/klaytn/pull/176/files/370c197b5a9e3b3ebe8641f630ecf975d090dfbd

Reply:
If a new version number is created, using const variable would be better than using the literal value, doesn't it?

Reply:
Fixed.